src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -91,6 +91,14 @@ class Config:
         "`streamer.put(token_ids)` and the streamer is responsible "
         "for any further processing.",
     )
+    callback: Optional[Callable[[Any], bool]] = Field(
+        default=None,
+        description="Callable that will be invoked "
+        "on each generated token. The callable must return a "
+        "Boolean value. If invocation returns `True`, the "
```

Comment:
Boolean value not required. Can return None for no effect

Reply:
Updated logic to only stop when `False` is returned